src/bgw/job.c

```diff
@@ -14,7 +14,7 @@
 #include "telemetry/telemetry.h"
 
 const char *job_type_names[_MAX_JOB_TYPE] = {
-	[JOB_TYPE_VERSION_CHECK] = "check_for_update_if_enabled",
+	[JOB_TYPE_VERSION_CHECK] = "telemetry_and_version_check_if_enabled",
```

Comment:
for my own edification, this is suppose to be read "telemetry_and_version_check, if_enabled" not "telemetry_and_version, check_if_enabled" right?